data-serving/data-service/test/model/data/location.minimal.json

```diff
@@ -1,4 +1,8 @@
 {
     "place": "Kings County Hospital Center",
```

Comment:
Isn't this invalid for lack of country?

Reply:
oops, good catch, thanks! was missing test for minimal case